nodes/AsyncFor.py

```diff
@@ -1,3 +1,29 @@
+"""
+AsyncFor astroid node
+
+This node iterates over async code with a for-loop like syntax. Asynchronous
+code doesn't wait for an operation to complete, rather the code executes all
+operations in one go. Only valid in body of an AsyncFunctionDef astroid node.
+
+Attributes:
+    - target  (Node(Name|Tuple|List))
```

Comment:
The classes `Name`, `Tuple`, and `List` are all subclasses of `Node`, so you don't need to write the "Node" explicitly in the type. Doing just `(Name | Tuple | List)` conveys the right information.